LayoutTests/svg/dom/SVGAnimationElement-begin-end-arguments-typechecks.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<script src="../../resources/js-test.js"></script>
+<svg>
+  <animate></animate>
+</svg>
+<script>
+description('Check that invalid values of arguments throw TypeError.');
+
+var animateElm = document.querySelector('animate');
```

Comment:
I think we do not like this kind of abbreviation for the variable name. Please use `animate`, `element` or `animateElement` instead.

Reply:
Done - changed to `animateElement`.